### ISSUE TITLE
This is a great project

### ISSUE THREAD

**Issue by yichengup** [yowipr/Manual#1]
The UI is very good, but I am not familiar with the functions yet, and it is not convenient enough.

**Comment by yowipr** [comment]
Manual is designed for the long term, betting that AI will be standard in the future.
there are still things to do and it is a long way, if you like you can specify how you currently use AI and what things you would add or change in Manual to make it comfortable for you to use, and I will try to do my best.

**Comment by yichengup** [comment]
> Manual is designed for the long term, betting that AI will be standard in the future. there are still things to do and it is a long way, if you like you can specify how you currently use AI and what things you would add or change in Manual to make it comfortable for you to use, and I will try to do my best.

I think your software is good overall, but there are many areas for improvement.
1. Move the entire canvas, scale the entire canvas, control the scaling, deformation and movement of the layer content, and switch with shortcut keys like PS.
2. The English UI interface can be translated into other languages, such as Chinese, so that the audience will increase.
3. I don't know where the layer mask is.
In short, your function can refer to Photoshop
